sdk/tests/conformance/glsl/misc/shader-uniform-packing-restrictions.html

```diff
@@ -262,6 +262,78 @@
       linkSuccess: true,
       passMsg: shaderType.type + " shader with " + minVars + " uniforms of " + info.type + " (the minimum required) should succeed",
     });
+    
+    if (maxPerRow > 1) {
+      
+      var generateCodeArray = function() {
+        var uniforms = [];
+        var sumTerms = [];
+        var itemcount = Math.max(Math.floor(shaderType.maxVectors / maxPerRow) + 1, minVars ); 
```

Comment:
"itemcount" should be computed once and its value shared when computing the message string below.